keyboards/kc60/config.h

```diff
@@ -15,8 +15,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#ifndef CONFIG_H
-#define CONFIG_H
+#pragma once
```

Comment:
Stupid Github won't let me comment on that line, but if you are already touching this file, what do you think about assigning a unique VID?

Reply:
71e287c